README.md

```diff
@@ -1 +1,34 @@
-# heat-map
\ No newline at end of file
+# DESIGN TEARDOWN
+
+This is a project that breaks apart the website for a popular design magazine – Smashing Magazine. 
+
+The goal for this exercise is to think in terms of visual hierarchy, typography and design principles. 
+
+## Sources
+
+You can check the original version here: https://www.smashingmagazine.com/
+
+This project is part of the Microverse Curriculum. www.microverse.org
+
+This project is part of the Odin Project Curriculum "PROJECT: DESIGN TEARDOWN" (you can find the complete project here:(https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown)
+
+
+## Live Version
+
+You can see it here:  https://euqueme.github.io/heat-map/
+
+## Built With
+
+HTML
+CSS
+Google Fonts
+Linters
+Stickler
+VSCode
+Atom
+
+## Authors
+
+Microverse students:
+@euqueme
+@majovainilla
```

Comment:
Nicely documented readme file :+1: 

Some links to improve the appearance of your readme files. A free course [here](https://www.udacity.com/course/writing-readmes--ud777) and some template for inspiration [here](https://github.com/othneildrew/Best-README-Template).